src/models/user.js

```diff
@@ -0,0 +1,31 @@
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db.js');
+
+const User = sequelize.define('User', {
+  id: {
+    type: DataTypes.UUID,
+    primaryKey: true,
+    defaultValue: DataTypes.UUIDV4,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  activationToken: {
+    type: DataTypes.STRING,
+  },
+  resetToken: {
+    type: DataTypes.STRING,
```

Comment:
Consider adding constraints or validations for `activationToken` and `resetToken` if they have specific formats or requirements. This can help prevent invalid data from being stored.